src/components/SegmentedControl/types.ts

```diff
@@ -0,0 +1,7 @@
+import { ReactNode } from 'react';
+
+export type OptionProps = {
+    label?: ReactNode;
+    value?: string | null | undefined;
```

Comment:
In what case value is null?
Also, when you set value to optional using `?` already you are allowing it to be undefined

Reply:
Hi @LeandroTorresSicilia has to be null as well due to the shape proptype definition expecting an object not just undefined and since this is optional we need to include the null type otherwise will warning about it

Reply:
The shape is defined as:
```
PropTypes.shape({
            label: PropTypes.node,
            value: PropTypes.string,
            icon: PropTypes.node,
        })
```
where value is not required and it is a string, in what case it will show a warning?
proptypes and typescript definitions must match